File: SoloLearn Codes/Roadrunner/Roadrunner.java
import java.util.Scanner;

public class Roadrunner
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int safety = sc.nextInt();
        int runner = sc.nextInt();
        int coyote = sc.nextInt();
        System.out.println(safety / runner < (safety + 50) / coyote ? "Meep Meep" : "Yum");
    }
}

File: SoloLearn Codes/Roadrunner/Roadrunner.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int safety = Convert.ToInt32(Console.ReadLine());
            int runner = Convert.ToInt32(Console.ReadLine());
            int coyote = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(safety / runner < (safety + 50) / coyote ? "Meep Meep" : "Yum");
        }
    }
}
